docs/specs/DB_SCHEMA.md

```
@@ -7,11 +7,12 @@ BookHub の Supabase PostgreSQL スキーマ定義。全テーブルは Row Leve
 ## Table of Contents
 
 1. [profiles](#profiles-ユーザープロフィール)
-2. [books](#books-書籍マスタ)
-3. [user_books](#user_books-ユーザー所持情報)
-4. [Migration History](#migration-history)
-5. [Row Level Security (RLS)](#row-level-security)
-6. [Query Examples](#query-examples)
+2. [series](#series-シリーズマスタ)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)